### ISSUE TITLE
Luminea Home Control Tür- und Fensteralarm für ZigBee-Gateway, App, Sprachsteuerung

### ISSUE THREAD

**Issue by reichemn** [dresden-elektronik/deconz-rest-plugin#6632]
## Device

- Product name: Luminea Home Control Tür- und Fensteralarm für ZigBee-Gateway, App, Sprachsteuerung
- Manufacturer: _TZ3000_4ugnzsli
- Model identifier: TS0203
- Device type :
  - Sensor

![image](https://user-images.githubusercontent.com/15368859/210010253-32d2efad-b1a6-4bc3-93eb-5a58fa73286b.png)

### Basic
![image](https://user-images.githubusercontent.com/15368859/210009785-4da4c7b7-4eed-4c32-a401-41cc4fe2270a.png)

### Identify
![image](https://user-images.githubusercontent.com/15368859/210010022-1362bd32-a58f-4394-986f-934119120584.png)

### IAS Zone
![image](https://user-images.githubusercontent.com/15368859/210009887-34bc0137-a575-4d01-853e-4f48b2b39284.png)

### OTAU
![image](https://user-images.githubusercontent.com/15368859/210009917-0894623a-1487-4c55-bfe9-7f9badd40f78.png)

### Power Configuration
![image](https://user-images.githubusercontent.com/15368859/210010010-a0e5c49f-9c0c-4f36-a50f-322bfab6766e.png)

**Comment by BabaIsYou** [comment]
May we can start with this [DDF](https://github.com/dresden-elektronik/deconz-rest-plugin/blob/master/devices/tuya/_TZ3000_n2egfsli_door_sensor.json) after chnaging lines 2 and 3 by these : 
```
  "manufacturername": ["_TZ3000_n2egfsli", "_TZ3000_oxslv1c9", "_TZ3000_7tbsruql", "_TZ3000_7d8yme6f", "_TZ3000_rgchmad8", "_TZ3000_bzxlofth","_TZ3000_4ugnzsli"],
  "modelid": ["TS0203", "TS0203", "TS0203", "TS0203", "TS0203", "TS0203", "TS0203"],
```

**Comment by reichemn** [comment]
I tested the mentioned modified DDF. The device is now recognized as door sensor in phoscon. Thank you!

**Comment by wunni1** [comment]
Hello, I want to use the same sensors. Are they still working with phoscon? Or should I do an update? Current version of phoscon (runs in HomeAssistant): 2.20.01 / 19.9.2022; Firmware: 26720700

**Comment by BabaIsYou** [comment]
@wunni1 you have to do the same as @reichemn did until the #6693 PR will be added in a next release.

**Comment by wunni1** [comment]
How can I modify the DDF in Home Assistant. I was able to start the DDF Editor. But I can't add the new file. When will the next release published?

**Comment by BabaIsYou** [comment]
> How can I modify the DDF in Home Assistant.

Think it could help [https://github.com/dresden-elektronik/deconz-rest-plugin/issues/6697#issuecomment-1404069986](url)

**Comment by wunni1** [comment]
Is this Sensor (TZ3000_4ugnzsli) now implemented  in the last release 2.21.0?
